app/containers/HomePage/index.js

```diff
@@ -22,9 +22,15 @@ class HomePage extends React.Component {
     this.onChangeRoute = this.onChangeRoute.bind(this);
     this.changeRouteToReadme = this.changeRouteToReadme.bind(this);
   }
+
+  shouldComponentUpdate(nextState, prevState) {
```

Comment:
No idea what my brain was thinking there :+1:

Not sure this can be faster than `(nextProps.ownerName !== this.props.ownerName || nextProps.projectName !== this.props.projectName)`, but then I don't know how it works internally.